cumulus/polkadot-parachain/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "polkadot-parachain-bin"
-version = "4.0.0"
+version = "5.0.0"
```

Comment:
I think mentioning it in the prdoc is enough. These numbers here are then automatically bumped as aggregation of all the prdocs.